L1Trigger/L1TCaloLayer1/plugins/L1TCaloSummary.cc

```diff
@@ -109,11 +109,9 @@ class L1TCaloSummary : public edm::stream::EDProducer<> {
 
   UCTLayer1* layer1;
 
-  hls4mlEmulator::ModelLoader loader;
-  std::shared_ptr<hls4mlEmulator::Model> model;
-
   bool overwriteWithTestPatterns;
   std::vector<edm::ParameterSet> testPatterns;
+  std::string modelName;
```

Comment:
Please rename all data members to follow CMS conventions: [2.9](Start data member names with lower case. A trailing “_” is the preferred method to distinguish a data member from the getter method (e.g. momentum_).):

> Start data member names with lower case. A trailing `_` is the preferred method to distinguish a data member from the getter method (e.g. `momentum_`).